docs/jwst/emicorr/arguments.rst

```diff
@@ -25,3 +25,6 @@ The ``emicorr`` step has the following step-specific arguments.
     This is to tell the code to do correction for the frequencies in
     the list with a reference file created on-the-fly instead of CRDS.
 
+``--nints_to_phase`` (integer, default=None)
```

Comment:
Isn't the parameter that emicorr_step takes called use_n_cycles, not nints_to_phase?

Reply:
the user can use either parameter, provide a number of integrations to phase or a number of cycles to use for finding the phase (which under the hood will calculate how many integrations are needed to complete that many cycles, the default is 3)

Reply:
`nints_to_phase` is already listed up in line 5 of this module. The new param added below in emicorr_step.py is "use_n_cycles". So the entry here should also be `use_n_cycles`, as David pointed out.